### PR TITLE
Update macOS build notes

### PR DESCRIPTION
**Pull request by merc1er**

### CODE REVIEW

File: contrib/osx/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This guide explains how to build Electron Cash binaries for macOS systems.
 We build our binaries on El Capitan (10.11.6) as building it on High Sierra
 makes the binaries incompatible with older versions.
 
-This assumes that the Xcode + Xcode Command Line tools (and thus git) are already installed. You can install older (and newer!) versions of Xcode from Apple provided you have a devloper account [from the Apple developer downloads site](https://developer.apple.com/download/more/).
+This assumes that the Xcode Command Line tools (and thus git) are already installed. You can install older (and newer!) versions of Xcode from Apple provided you have a devloper account [from the Apple developer downloads site](https://developer.apple.com/download/more/).
```
Comment (cculianu) [inline contrib/osx/README.md:10]: You cannot install Xcode Command Line tools without installing Xcode.  I am unsure for the reason behind this edit....?

File: contrib/osx/README.md [hunk 2]
```diff
@@ -16,14 +16,21 @@ This assumes that the Xcode + Xcode Command Line tools (and thus git) are alread
 
 The above ensures that you pull in the zbar, secp256k1, and other submodules.
 
-## 2. Use the provided script to begin building.
+## 2. Make sure coreutils is installed
+
+With [brew](https://brew.sh) installed, run
```
Comment (cculianu) [inline contrib/osx/README.md:21]: Two things:

1.  I believe brew is terrible and merits to die as a project.  Please also add MacPorts instructions while we are here:

`sudo port install coreutils`

2. Also pyenv is not needed for building this project -- the build script automatically installs pyenv for you. Please delete that line.

File: contrib/osx/README.md (hunk 1 above)
Comment (merc1er) [reply contrib/osx/README.md:10]: You can. I have the command line tools with Xcode (which is like 10GB big).

To do this, simply run the command `xcode-select --install`.

[Here is a tutorial](https://www.maketecheasier.com/install-command-line-tools-without-xcode/).

File: contrib/osx/README.md (hunk 2 above)
Comment (merc1er) [reply contrib/osx/README.md:21]: I tried to run this  on Big Sur and for some reason the script failed with something like "pyenv is not installed, aborting...". 

This is why I added this line. Please confirm one more time that you don't want this line and I'll remove it :)

Comment (cculianu) [reply contrib/osx/README.md:21]: Holy cow! That was NOT possible before AFAIK.  That's awesome.  TIL.

File: contrib/osx/README.md (hunk 1 above)
Comment (cculianu) [reply contrib/osx/README.md:10]: This I did not know.  Thanks.  TIL.  Resolving.   Thanks!

File: contrib/osx/README.md (hunk 2 above)
Comment (cculianu) [reply contrib/osx/README.md:21]: See -- this is why I don't use brew. :)

Well -- then can we clarify that's a brew-specific thing?  Not everybody uses brew.  I certainly don't -- Josh Elliithorpe doesn't.. there are plenty of people  not using it...

Maybe make the `pyenn` line a brew-specific: e.g. "If on brew do this...", "If on macports do this...".

Comment (merc1er) [reply contrib/osx/README.md]: Added

**Comment by cculianu** [comment]
Looks great -- thanks for the update!

**Comment by cculianu** [comment]
Looks great -- thanks for the update!

**Comment by cculianu** [comment]
Looks great -- thanks for the update!

**Pull request merged**